drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/controller/AnalyzerControllerImplTest.java

```diff
@@ -76,6 +106,50 @@ public void areHandlersAreSetUp() throws Exception {
     }
 
     @Test
+    public void areHandlersAreTornDownOnTerminate() throws
```

Comment:
If I was to be picky I'd say this test name should more correctly be "areHandlersTornDownOnTerminate"